.env.example

```diff
@@ -78,3 +77,18 @@ DOCKER_HOST_USER_ID=1000
 LLM_API_KEY=
 LLM_MODEL=
 LLM_ENDPOINT=
+
+# SPATIE MEDIA LIBRARY
+MEDIA_DISK=public
+QUEUE_CONNECTION=redis
+MEDIA_QUEUE=
+QUEUE_CONVERSIONS_BY_DEFAULT=true
+QUEUE_CONVERSIONS_FOR_COLLECTIONS=true
+IMAGE_DRIVER=gd
+FFMPEG_PATH=/usr/bin/ffmpeg
+FFPROBE_PATH=/usr/bin/ffprobe
+MEDIA_DOWNLOADER_SSL=true
+MEDIA_TEMPORARY_URL_DEFAULT_LIFETIME=5
+ENABLE_MEDIA_CDN=false
+MEDIA_PREFIX=
+FORCE_HTTPS_IN_MEDIA_URLS=true
```

Comment:
do u need that stuff in `.env`?
all ot of from this should be not changed - I think